### PR TITLE
[Migrate Test to TPUVM] Modify all test config to enable TPUVM

### PR DESCRIPTION
**Pull request by ManfeiBai**
modify all the libsonnet files under "tests/pytorch/nightly":
- adding `tpuVm`;
- update all config in `configs` with `tpuVm`.

### CODE REVIEW

File: tests/pytorch/nightly/cpp-ops.libsonnet [hunk 1]
```diff
@@ -54,7 +68,7 @@ local utils = import 'templates/utils.libsonnet';
   },
 
   configs: [
-    operations + v2_8 + common.Functional + timeouts.Hours(4),
+    operations + v2_8 + common.Functional + timeouts.Hours(4) + tpuVm,
```
Comment (ManfeiBai) [inline tests/pytorch/nightly/cpp-ops.libsonnet:71]: according to the below comments, there are some different way to enable TPUVM, we might need to check with this bug and author for more details

**Review by wonjoolee95 — COMMENTED** [review]
Thanks! LGTM overall. Seems like we want to disable some of these new TPUVM tests, based on https://docs.google.com/spreadsheets/d/12_fLEsJnmfN9aLXkzpXUKCl0KTWgoMX6rK19Q2XEKlE/edit?resourcekey=0-1mrwODxoOYa-Aul7jD_ssw#gid=0. Do you want to include that change here or are we planning to follow-up as a separate PR?

File: tests/pytorch/nightly/cpp-ops.libsonnet (hunk 1 above)
Comment (ManfeiBai) [reply tests/pytorch/nightly/cpp-ops.libsonnet:56]: checked and deprecated the commented used tpuvm,
decide to use the `tpuVm` like other tests

**Comment by ManfeiBai** [comment]
> Thanks! LGTM overall. Seems like we want to disable some of these new TPUVM tests, based on https://docs.google.com/spreadsheets/d/12_fLEsJnmfN9aLXkzpXUKCl0KTWgoMX6rK19Q2XEKlE/edit?resourcekey=0-1mrwODxoOYa-Aul7jD_ssw#gid=0. Do you want to include that change here or are we planning to follow-up as a separate PR?

done as a seperate PR: https://github.com/GoogleCloudPlatform/ml-testing-accelerators/pull/779

**Comment by wonjoolee95** [comment]
Thanks for disabling the tests. Now we need to rebase this PR and should be good to go.

**Comment by ManfeiBai** [comment]
> Thanks for disabling the tests. Now we need to rebase this PR and should be good to go.

Thanks, rebased

**Review by wonjoolee95 — APPROVED** [review]

File: tests/pytorch/nightly/mnist.libsonnet [hunk 2]
```diff
@@ -98,7 +98,6 @@ local utils = import 'templates/utils.libsonnet';
   },
 
   configs: [
-    mnist + convergence + v2_8 + timeouts.Hours(1) + mixins.Experimental,
```
Comment (JackCaoG) [inline tests/pytorch/nightly/mnist.libsonnet:101]: let's leave this test in 2vm so we get some signal on 2vm.

Comment (ManfeiBai) [reply tests/pytorch/nightly/mnist.libsonnet]: Thanks, Jack, kept this test

**Review by JackCaoG — APPROVED** [review]

**Pull request merged**